lib/parse/data/parseEmbedData.ts

```diff
@@ -69,8 +78,6 @@ const parseEmbedData = (config: IEmbedConfig, rssData?: IRss): IEmbedData => {
   const followUrls = {
     ...((subscribeUrl || feedUrl) && { rss: subscribeUrl || feedUrl })
   };
-  const paymentPointer =
-    process.env.NODE_ENV === 'development' && process.env.PAYMENT_POINTER;
```

Comment:
I would keep this around to allow devs the ability to define a dummy pointer during development. Something like `(process.env.NODE_ENV === 'development' && process.env.PAYMENT_POINTER) ||  podcastValueRecipient?.address` should work to fallback to what was in the feed if the dev hasn't defined a pointer.

Reply:
with the updates to now use the value from the RSS feed, why wouldn't you just use test feed that includes a pointer in podcast:value?

Reply:
done!